app/dao/services_dao.py

```diff
@@ -282,7 +284,12 @@ def dao_create_service(
     service.id = service_id or uuid.uuid4()  # must be set now so version history model can use same id
     service.active = True
     service.research_mode = False
-    service.crown = service.organisation_type == 'central'
+    if organisation:
+        service.crown = organisation.crown
+    elif service.organisation_type in CROWN_ORGANISATION_TYPES:
+        service.crown = True
+    elif service.organisation_type in (NON_CROWN_ORGANISATION_TYPES + ['nhs']):
```

Comment:
why not just add nhs to the list?

Reply:
I don't think we do, since we don't have a constraint on the table at this point, and in next PR I will migrate all 'nhs' organisation_type organisations to be either 'nhs_local' or 'nhs_central' before introducing the constraint.

Reply:
We could temporarily add it, that's true (in this PR, which is a draft right now, we will be getting rid of generic 'nhs' type: https://github.com/alphagov/notifications-api/pull/2565 )

Reply:
Actually you are right that this is cleaner code, I added it.